### DISCUSSION TITLE
See TLDR on article card

### DISCUSSION THREAD

**Discussion by AnishDe12020** [dailydotdev/daily#437] [Feature Requests]
AI-generated TLDRs is an awesome feature, but they are on the post page of the article, something people usually don't visit unless they are commenting. The TLDRs can be added directly to the card (that is the article cards on the feed). There can be sort of an accordion or something so that it doesn't take up much space. Maybe also a toggle can be added under preferences to show/hide it on the article card.

**Comment by rebelchris** [comment]
Hi Anish,

Great request and we are working on something very similar.
 It's currently in the spec phase, so keep your eyes out for this feature 👀

**Reply by AnishDe12020** [reply]
Cool, can't wait for it!!!
